entity-framework/core/modeling/shadow-properties.md

```diff
@@ -44,11 +46,19 @@ var blogs = context.Blogs
 
 Shadow properties cannot be accessed after a no-tracking query since the entities returned are not tracked by the change tracker.
 
+## Configuring indexer properties
+
+You can use the Fluent API to configure indexer properties. Once you've called the string `IndexerProperty`, you can chain any of the configuration calls you would for other properties. In the following sample, `Blog` has indexer defined and it will be used to create indexer property.
+
+[!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/IndexerProperty.cs?name=ShadowProperty&highlight=3)]
+
+If the name supplied to the `IndexerProperty` method matches the name of an existing indexer property, then the code will configure that existing property. If the entity type has a property, which is backed by a property on the entity class, then an exception is thrown since indexer properties must be accessed via indexer only.
```

Comment:
```suggestion
If the name supplied to the `IndexerProperty` method matches the name of an existing indexer property, then the code will configure that existing property. If the entity type has a property, which is backed by a property on the entity class, then an exception is thrown since indexer properties must only be accessed via the indexer.
```